component.json

```diff
@@ -66,11 +68,93 @@
       },
       "metadata": {
         "in": {
-            "type": "object",
-            "properties": {}
+          "type": "object",
+          "properties": {}
+        },
+        "out": {}
+      }
+    },
+    "write_attachment_from_json": {
+      "main": "./lib/actions/writeFromJson.js",
+      "title": "Write CSV attachment from JSON Object",
+      "help": {
+        "description": "Multiple incoming events can be combined into one CSV file with the write CSV action. See <a href='https://github.com/elasticio/csv-component/'>link</a> for additional documentation.",
```

Comment:
https://docs.elastic.io/guides/configuring-help-links.html

Reply:
Ok, got it. I'll remove second link